library/Rules/CountryCode.php

```diff
@@ -1,41 +1,318 @@
 <?php
 
-/*
- * This file is part of Respect/Validation.
- *
- * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
- *
- * For the full copyright and license information, please view the "LICENSE.md"
- * file that was distributed with this source code.
- */
-
```

Comment:
I copied from [this file](https://github.com/Respect/ValidationLocale/blob/master/Rules/Country.php) eheh.

No problem, I will fix it.